Extension/src/nativeStrings.json

```diff
@@ -232,5 +232,6 @@
     "inline_macro": {
         "text": "Inline macro",
         "hint": "'Inline' is a command and not an adjective, i.e. like 'Expand macro'."
-    }
+    },
+    "unable_to_access_browse_database": "Unable to access browse database. ({0})"
```

Comment:
maybe use "...database in {0}." instead?

Reply:
I'd kinda prefer to keep it this way, so we don't have to worry about how translators would treat it within a sentence.

Reply:
I was thinking it would be better to make it a real sentence or the translators may be get confused by the ". ()".

Reply:
The logging is mostly for us to track where it failed to access the database, so putting `{0}` in ( ) is sufficient.